README.md

```diff
@@ -72,14 +72,21 @@ Valvat::Lookup.validate("DE345789003")
 # => true or false or nil
 ```
 
-To keep backwards compatibility lookups of UK VAT numbers against the HMRC API are only performed with the option `:uk` set to true.
+To keep backwards compatibility lookups of UK VAT numbers against the HMRC API are only performed with the option `:uk` with Authentication credentials.
 
 ```ruby
-Valvat::Lookup.validate("GB553557881", uk: true)
-# => true or false or nil
+Valvat::Lookup.validate(
+  "GB553557881",
+  uk: {
+    live: true,
+    client_id: '<client_id>',
+    client_secret: '<client_secret>'
+  }
+)
+# => Hash ({ client_id: '...', client_secret: '...' }) or false or nil
```

Comment:
Are these response examples valid?

Reply:
Yes, because it's about `options` for method (not global config), so for example when you pass only `client_id` & `client_secret` except `live`, the `live` config will be fetched from global configuration

Reply:
of course if globally uk is not set as false or nil